pkg/trace/api/ratelimiter_test.go

```diff
@@ -131,29 +131,29 @@ func TestRateLimiterPermits(t *testing.T) {
 
 	ps := newRateLimiter()
 	ps.SetTargetRate(0.2)
-	assert.Equal(0.2, ps.RealRate(), "by default, RealRate returns wished rate")
-	assert.True(ps.Permits(100), "always accept first payload")
+	assert.Equal(1., ps.RealRate(), "by default, RealRate returns 1")
+	assert.False(ps.Permits(100), "always accept first payload")
```

Comment:
^ either fix the comment or address the comment above. But something is wrong here, the comment is not matching the action (says payload is accepted, but Permits() returns false)